android/app/src/main/java/com/emergetools/hackernews/ui/preview/AppStoreSnapshot.kt

```diff
@@ -3,9 +3,28 @@ package com.emergetools.hackernews.ui.preview
 import android.content.res.Configuration.UI_MODE_NIGHT_YES
 import androidx.compose.ui.tooling.preview.Devices
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewLightDark
 import com.emergetools.snapshots.annotations.EmergeAppStoreSnapshot
 
 @EmergeAppStoreSnapshot
 @Preview(device = Devices.PIXEL_5, uiMode = UI_MODE_NIGHT_YES)
 @Preview(device = Devices.PIXEL_5)
 annotation class AppStoreSnapshot
+
+@Preview(
```

Comment:
Seems we're having some trouble with some of these. Any chance you could use a default device - maybe skip the landscape temporarily and do:
`device = Devices.PIXEL_FOLD`
and
`device = Devices.PIXEL_TABLET`

Reply:
Done